lib/main.js

```diff
@@ -114,9 +115,9 @@ export default {
         }
 
         // The file is added to the arguments last.
-        args.push(file);
-        return helpers.exec(command, args, {stream: "stderr"}).then(output =>
-          helpers.parse(output, regex)
+        args.push("-");
+        return helpers.exec(command, args, {stdin: text, stream: "stderr"}).then(output =>
+          helpers.parse(output, regex, {filePath : filePath})
```

Comment:
It is Javascript, and sorry, that was a roundabout way of saying: "This would look better simplified to `{ filePath }`" 😛